scripts/datenlord_metrics_test.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/sh
+
+set -o errexit
+set -o nounset
+set -o xtrace
+
+# Need to wait for some time before metrics are collected by Promethues
+promethues_wait_time=5 
+
+kubectl apply -f $DATENLORD_METRICS_TEST
+kubectl wait --for=condition=Ready pod mysql-datenlord-test --timeout=60s
+kubectl exec -i mysql-datenlord-test -- bash -c "echo test > /usr/share/nginx/html/testfile"
+kubectl exec -i mysql-datenlord-test -- bash -c "cat /usr/share/nginx/html/testfile"
+sleep $promethues_wait_time
```

Comment:
use `PROMETHUES_WAIT_TIME` for shell env var

Reply:
Done